PocketCaddy/PocketCaddy/MainView/LaunchView.swift

```diff
@@ -4,27 +4,28 @@
 //
 //  Created by Byeon jinha on 2022/06/10.
 //
-
+import Combine
+import Foundation
 import SpriteKit
 import SwiftUI
 
 class MyTimer: ObservableObject {
```

Comment:
```suggestion
final class MyTimer: ObservableObject {
```

final에 대해 아십니까? (도를 아십니까 말투)

Reply:
final 을 넣으면 대략 오버라이딩이 안되는 거로 알고 있어요. 안엔 내용이 중요할 때? 쓰는 거로 알고 있습니다.

Reply:
오오오 !!!! 그렇다면 타이머도 적용 어떨까요 ???!

Reply:
좋습니다!! 적용하겠습니다!